src/box/memtx_rtree.cc

```diff
@@ -146,20 +146,139 @@ extract_rectangle(struct rtree_rect *rect, struct tuple *tuple,
 /* {{{ MemtxRTree Iterators ****************************************/
 
 struct index_rtree_iterator {
-        struct iterator base;
-        struct rtree_iterator impl;
+	struct iterator base;
+	struct rtree_iterator impl;
 	/** Memory pool the iterator was allocated from. */
 	struct mempool *pool;
+	/* last fetched tuple (pos). Is used for pagination */
+	struct tuple *last;
 };
 
+/* Min distance of last saved tuple.
```

Comment:
Please, fix this comment and all the other ones. How we write them:
One-liners:
```c
/** The comment. */
```

Multi-liners:
```c
/**
 * Multi line
 * comment.
 */
```